### PR TITLE
[release-1.4] Sync to installer charts 

### PR DESCRIPTION
**Pull request by incfly**
Need to refer to another SHA in installer 1.4 branch, plus manually modify the `default.yaml`. Only these two things are needed for 1.4 release.

And the update the golden files.

https://github.com/istio/istio/issues/14524

### CODE REVIEW

**Comment by incfly** [comment]
@ostromart mind approving this one for 1.4 as well?

**Review by ostromart — APPROVED** [review]

**Comment by ostromart** [comment]
@incfly this also needs release manager approval, ping one of them to get this moving.

**Comment by incfly** [comment]
Thanks for reminding actually in today's networking WG meeting, we decide not make it on by default, so closing this PR now.

**Pull request closed without merging**
